jwt-payload.json.example

```diff
@@ -1,4 +1,5 @@
 {
+    "organization_id": "hyphacoop",
```

Comment:
@benhylau FYI this is my proposed extension to our JWT format so far. Let me know what you think.

Reply:
Just need to know if we can use `-`. Either way I think it's okay, but having `-` may be easier to read.

Reply:
Yes, we can use `-` or `_`. In general, anything that you'd put in a directory or file name.